src/backend/core/entitlements/resolvers/entitlement_resolver.py

```
@@ -0,0 +1,180 @@
+        This method also return in the object the last level at which the entitlement was compliant or not via
+        "_reason_level" key.
+
+        Example:
+        {
+            "can_store": True,
+            "can_store_resolve_level": "user_override",
+        }
+        """
+
+        entitlements = get_entitlements_by_priority(context["entitlements"])
+        print('entitlements', entitlements)
```

Ruff (0.14.8)

`print` found

Remove `print`

(T201)